core/components/gitpackagemanagement/processors/mgr/gitpackage/update.class.php

```diff
@@ -193,14 +193,15 @@ private function updateExtensionPackage() {
         if($extPackage !== false){
             $modelPath = $this->packagePath . $this->object->dir_name . "/core/components/" . $this->newConfig->getLowCaseName() . "/" . 'model/';
             $modelPath = str_replace('\\', '/', $modelPath);
-            if($extPackage === true){
-                $this->modx->addExtensionPackage($this->newConfig->getLowCaseName(),$modelPath);
-            }else{
-                $this->modx->addExtensionPackage($this->newConfig->getLowCaseName(),$modelPath, array(
-                      'serviceName' => $extPackage['serviceName'],
-                      'serviceClass' => $extPackage['serviceClass']
```

Comment:
serviceName and serviceClass are missing after this update, they should still be present if they are set